decorators/__tests__/autoHide-test.js

```diff
@@ -27,7 +27,7 @@ describe('autoHideContainer', () => {
   });
 
   function render(props = {}) {
-    var AutoHide = autoHideContainer(<span />);
+    var AutoHide = autoHideContainer('span');
```

Comment:
I am not sure what we are testing here, the signature of autoHideContainer is:

```js
function autoHideContainer(ComposedComponent) {
```

and the render:

```js
return <ComposedComponent {...this.props} />;
```

So yes we fixed the warnings but we have another issue I feel here